pipelines/main/platforms/build_linux.yml

```diff
@@ -27,10 +27,33 @@ steps:
           # export JULIA_CPU_TARGET, which is picked up by the build system
           ARCH="$${ARCH??}" source .buildkite/utilities/export_julia_cpu_target.sh
 
+          echo "--- Create the Make.user file"
+          rm -f Make.user.tmp
+          echo "${MAKE_FLAGS?}" | tr "," "\n" > Make.user.tmp
+          if [[ "$${BUILDKITE_PIPELINE_SLUG:?}" == "julia-master" ]]
+          then
+            banner="Official https://julialang.org/ release"
+            echo "TAGGED_RELEASE_BANNER=\"$${banner:?}\"" >> Make.user.tmp
+          elif [[ "$${BUILDKITE_PIPELINE_SLUG:?}" =~ ^julia-release- ]]
+          then
+            banner="Official https://julialang.org/ release"
+            echo "TAGGED_RELEASE_BANNER=\"$${banner:?}\"" >> Make.user.tmp
+          elif [[ "$${BUILDKITE_PIPELINE_SLUG:?}" == "julia-buildkite" ]]
+          then
+            banner="Not an official binary; built as part of the CI infrastructure testing"
+            echo "TAGGED_RELEASE_BANNER=\"$${banner:?}\"" >> Make.user.tmp
+          else
+            :
+          fi
+          echo "VERBOSE=1" >> Make.user.tmp
+          rm -f Make.user
+          awk NF Make.user.tmp > Make.user
+          rm -f Make.user.tmp
+          wc -l Make.user
+          cat Make.user
```

Comment:
I'm confused, why are you doing this?  Why not just write out to `Make.user` directly?

Reply:
I wanted to remove blank lines, and I couldn't figure out how to do that in place.

Reply:
I..... I don't think it's that important.  😂